forte/models/da_rl/magic_model.py

```diff
@@ -21,8 +21,16 @@
 
 import copy
 from typing import Dict
-import torch
-from torch import nn
+
+try:
+    import torch
+    from torch import nn
+except ImportError as e:
+    raise ImportError(
+        " `pytorch` is not installed correctly."
+        " Please refer to [extra requirement for models](pip install forte[models])"
```

Comment:
For all these exception messages, it would be best to:
1. Mention that PyTorch needed to be installed, 
2. Also mention the extra requirements.

So sometimes the user can simply follow the error message to install pytorch, and don't need to go to read the documentation